packaging/rosdep.yaml

```diff
@@ -0,0 +1,4 @@
+fog_msgs:
+    ubuntu: ros-ROS_DISTRO-fog-msgs=0.0.8*
+dynamicEDT3D:
+    ubuntu: ros-ROS_DISTRO-dynamic-edt-3d
```

Comment:
As far as I know, the octomap_server doesn't have any dependencies. These are for the navigation node. Of course it won't affect the installation process, as Rosdep will not try to install those. But might be deleted also to have cleaner repo. This rosdep.yaml can be deleted.